libs/api/database/src/lib/content/stores/content.store.ts

```diff
@@ -131,47 +118,44 @@ export class ContentStore {
     }
 
     /**
-     * Fetches all published documents based on kind, limited by page number.
+     * Fetches a section by ID. Performs an extra check to only fetch a published section.
```

Comment:
nit: this makes it sounds like the check is always performed, when it's an argument